src/backend/marsha/core/models/file.py

```diff
@@ -64,6 +64,12 @@ def is_ready_to_show(self):
         """
         return self.uploaded_on is not None
 
+    def uploaded_on_stamp(self):
```

Comment:
Could you add tests on this new method ?

And IMO you are changing the behavior already implemented. Never the timestamp is computed when the `uploaded_on` is not set.

Reply:
In which case of this commit do we compute a timestamp if `updloaded_on` doesn't exists ?
Here, we only extract `to_timestamp(self.uploaded_on)` in his own function.